docassemble/MAVirtualCourt/data/questions/basic-questions.yml

```diff
@@ -822,8 +828,8 @@ subquestion: |
   There was a problem delivering this to the ${courts[0]}. We couldn't
   locate the contact information for this court.
   % endif
-
-  Click "Next" if you're ready to send it.
+ 
+  Press below to deliver the form.
```

Comment:
That still implies to me that the button will perform the action of sending the email.

Best I've come up with so far: 'Next you'll send the form to the court.'

Other ideas:
- 'Keep going to get ready to email the form.'
- 'Keep going to email the form.'
- 'Keep going to send the form to the court.'

Reply:
this does send the form.

Reply:
Ah, getting sending pages mixed up.